profile/src/main/res/layout/feedback_question1.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:card_view="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="fill_parent"
+    android:layout_height="wrap_content"
+    android:animateLayoutChanges="true"
+    card_view:cardBackgroundColor="@color/colorWhite"
+    card_view:cardCornerRadius="0dp"
+    card_view:cardElevation="0dp">
+
+    <FrameLayout
+        android:id="@+id/verticalAccent1"
+        android:layout_width="3dp"
+        android:layout_height="match_parent"
+        android:background="@color/fightPandemicsNeonBlue"
+        android:visibility="gone"
```

Comment:
why different visibilities?

Reply:
It's just so it appears in the XML editor. The view model makes this appear and disappear.

Reply:
Should I remove it or is it fine?

Reply:
Its ok